docs/topics/templates/document-attributes.adoc

```diff
@@ -10,8 +10,22 @@
 // *****************
 // * Product names *
 // *****************
+ifdef::mta[]
 :ProductName: Migration Toolkit for Applications
+:DocInfoProductName: Migration Toolkit for Applications
+:DocInfoProductNumber: 5.3
 :ProductShortName: MTA
+:DocInfoProductNameURL: migration_toolkit_for_applications
+
+endif::[]
+ifdef::mtr[]
+:ProductName: Migration Toolkit for Runtimes
+:DocInfoProductName: Migration Toolkit for Runtimes
+:DocInfoProductNumber: 1.0
+:ProductShortName: MTR
+:DocInfoProductNameURL: migration_toolkit_for_runtimes
+endif::[]
+
 
 // *******************
 // * Component names *
```

Comment:
Plugin Name for MTR should be MTR Plugin

Reply:
IDEPluginFilename: migrationtoolkit-mta-eclipse-plugin-repository

PluginName: MTA plugin